baselines/dom.generated.d.ts

```diff
@@ -3232,13 +3232,13 @@ declare var CSSTransition: {
  * Available only in secure contexts.
  */
 interface Cache {
-    add(request: RequestInfo): Promise<void>;
+    add(request: RequestInfo | URL): Promise<void>;
     addAll(requests: RequestInfo[]): Promise<void>;
```

Comment:
Not sure how to treat this one 🤔 but okay for now.

Reply:
I'm not sure how to properly implement this in generator either.
But in my opinion, even something like `RequestInfo[] | URL[]` would be an okay trade-off here: mixing those in userland code should be rarely needed.

Another option is to refactor this later by introducing some `urlstring` and `RequestInfoOrUrl` typedefs. Dunno about downsides.

These should be affected as well:
https://github.com/microsoft/TypeScript-DOM-lib-generator/blob/d060ef578ae9193a7606833261196977e446731b/baselines/dom.iterable.generated.d.ts#L25-L27

Reply:
Hmmmmmm. Let's merge this and tackle this in a separate patch.